src/Database/PostgreSQL/PQTypes/Model/Index.hs

```diff
@@ -203,6 +213,7 @@ sqlCreateIndex_ concurrently tname idx@TableIndex{..} = mconcat [
   , if null idxInclude
     then ""
     else " INCLUDE (" <> mintercalate ", " idxInclude <> ")"
+  , if idxUnique && idxNotDistinctNulls then " NULLS NOT DISTINCT" else ""
```

Comment:
Can you put then/else in separate lines just like for the idxInclude above? It's more readable.